scripts/py_matter_yamltests/matter_yamltests/hooks.py

```diff
@@ -221,7 +221,9 @@ async def step_manual(self):
     def show_prompt(self,
                     msg: str,
                     placeholder: Optional[str] = None,
-                    default_value: Optional[str] = None) -> None:
+                    default_value: Optional[str] = None,
+                    endpoint_id: Optional[int] = None,
```

Comment:
This hooks object is owned by the test harness. In theory adding an additional optional member should be fine, but would like input from the TH team re: how they're using this.

@fabiowmm @ccruzagralopes

Reply:
@rquidute please take a look here.

Reply:
From TH perspective we won't handle endpoint argument, so it may not be necessary to be included as argument at show prompt hook method.